components/lib/biometric-prompt/src/main/java/mozilla.components.lib.auth/BiometricPromptFeature.kt

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.lib.auth
+
+import android.content.Context
+import androidx.annotation.VisibleForTesting
+import androidx.biometric.BiometricManager.Authenticators.BIOMETRIC_WEAK
+import androidx.biometric.BiometricManager.Authenticators.DEVICE_CREDENTIAL
+import androidx.biometric.BiometricPrompt
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import mozilla.components.support.base.feature.LifecycleAwareFeature
+import mozilla.components.support.base.log.logger.Logger
+
+/**
+ * A [LifecycleAwareFeature] for the Android Biometric API to prompt for user authentication.
+ *
+ * @param context Android context.
+ * @param fragment The fragment on which this feature will live.
+ * @param authenticationCallbacks Callbacks for BiometricPrompt.
+ */
+class BiometricPromptFeature(
```

Comment:
The path to this file seems wrong. Here's an example of the src/main/java folder in lib-state:

https://github.com/mozilla-mobile/android-components/tree/main/components/lib/state/src/main/java/mozilla/components/lib/state

Here it seems to be:
.../lib/biometric-prompt/src/main/java/mozilla.components.lib.auth/ (should be subfolders?)

Reply:
Done